packages/entity-list/src/components/SearchFilterList/SearchFilterList.js

```diff
@@ -22,16 +32,32 @@ const SearchFilterButton = ({setActive, active, label}) =>
         e.stopPropagation()
       }}
       icon={active ? 'minus' : 'plus'}
-      dense />
+      dense/>
+    <StyledMenuWrapper active={active}>
+      <BallMenu buttonProps={{icon: 'ellipsis-h'}}>
```

Comment:
Failed prop type: Invalid prop `children` supplied to `BallMenu`. If there is no MenuItem (canEdit=false & canDelete=false)